### ISSUE TITLE
investigate accessibility: make hidden instead of off-screen

### ISSUE THREAD

**Issue by hchiam** [hchiam/minilink#8]
# Problem:
Screen readers will be able to "see" the off-screen `input` tag. For better usability for everyone, it should be hidden (or not used at all). This also eliminates the need to hit tab an extra time if you're navigating purely from the keyboard.

# Foreseeable difficulty in implementing this:
It still needs to be able to copy the URL to the clipboard. The `input` tag was used to begin with because copying the `innerHTML` of the `a` tag didn't seem to work. 
Relates to recent fix for #6.
